scormxblock/static/html/studio.html

```diff
@@ -79,6 +79,15 @@
       </div>
       <div class="setting-help" style="display:block">${block.fields['weight'].help}</div>        
     </li>
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label" >Locking</label>
+      </div>
+      <div class="setting-help" style="display:block">${block.fields['is_next_module_locked'].help}
```

Comment:
The checkbox on the right looks odd. Although the design in the ticket is at the right, we are using checkboxes on the left in scorm studio edit module as shown in the screen shot for auto completion.
Should we use the checkbox for gating at the left side too?
![gating_studio](https://user-images.githubusercontent.com/17109504/56112535-cfe13a00-5f74-11e9-9968-462b82de4fdb.png)

Reply:
If you see at the bottom, all options have check boxes at right, design looks good to me.

Reply:
I think @moeez96 is right. Checkbox should be on left with Locking heading like it is on the top for Enable completion upon viewing SCORM file.

Reply:
@jawad-khan I cant see any check boxes at right. Can you please attach a screen shot? Thanks

Reply:
Its in above screenshot too. For Display type iframe or popup we have radio buttons on the right of text.

Reply:
@jawad-khan Those are radio buttons. I'll clear this out with Janice no worries.

Reply:
@jawad-khan Confirmed from janice, please move the checkmark to the left of the heading `Locking`. Thanks!